db/migrate/20190418114431_add_approval_voting_fields.rb

```diff
@@ -0,0 +1,7 @@
+class AddApprovalVotingFields < ActiveRecord::Migration
```

Comment:
Not sure about this DB structure: 🤔 

I think the simplest would be to add all approval voting attributes to the `budgets` table. But if we want to allow for different number of votes in different `headings` or `groups` it would make sense to move these attributes to the `groups` or `headings` tables. Taking into account your requirements in NY, would it make sense to keep the `voting_style` and `voting_cant_exceed_heading_budget` in the `budgets` table and the `number_votes_per_heading` in the `budget_headings` table?

Reply:
I pinged Devin, he is ok with it. I will move the fields.

Reply:
@voodoorai2000 : I pinged @devinbalkind on this. The 'exceed' flag should be moved to Budget. In the case of voting t would be useful to have the ability to mark the number of votes per heading in a per group basis.

Reply:
@voodoorai2000 : sorry I missed the last part. I think that will be good. I will ask devin and set it up.